app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] In the user model you named the foreign key Author_id therefore you should mention this in the belongs to part of the relation as well like below:
```
belongs_to :author, class_name: 'User'